template/_gitignore

```diff
@@ -26,7 +26,7 @@ node_modules/
 !/web/wp-content/*/.gitkeep
 
 # Add exception for the project's theme and plugins
-!/web/wp-content/themes/<%= slug %>
+!/web/wp-content/themes/studiometa
```

Comment:
@studiometa/wordpress We have decided to use `studiometa` as the name of the theme. I'm wondering, why not use a default value `studiometa` and keep the ability to change it if needed!